### ISSUE TITLE
Añadir opción en "buscar biblias"

### ISSUE THREAD

**Issue by Cyb3rG0D7** [quelea-projection/Quelea#206]
Después de usar "buscar biblias", añadir una opción que se pueda enviar el versículo seleccionado a la lista.

**Comment by ArvidNy** [comment]
There is actually an button next to the search box to add the entire chapter to the schedule list. Is your feature request to only add the selected verse instead of the chapter?

**Comment by Cyb3rG0D7** [comment]
Sí, quiero que al encontrar el verso mandarlo a la lista para luego enviar a "en vivo", pero sería bueno tener las 2 opciones, el verso y el capítulo

**Comment by ArvidNy** [comment]
I see, then I'm changing the subject of this issue to reflect your request.
